src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -40,7 +47,8 @@ export const LocatorEditDialog: React.FC<Props> = ({
   elemName,
   elemText,
 }) => {
-  const [locatorValidity, setLocatorValidity] = useState<string>(validity?.locator || "");
+  const validationEnabled = useLocatorValidationEnabled();
```

Comment:
I offer to rename it to `isValidationEnabled`, because usually variables that contain boolean values starts with `is` or `has`

Reply:
sure